core/src/main/java/jenkins/views/JenkinsHeader.java

```diff
@@ -0,0 +1,38 @@
+package jenkins.views;
+
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import hudson.Extension;
+import hudson.ExtensionList;
+
+@Extension(ordinal = Integer.MIN_VALUE)
+public class JenkinsHeader implements Header {
+    
+    private static Logger LOGGER = Logger.getLogger(JenkinsHeader.class.getName());
+
+    @Override
+    public boolean isHeaderEnabled() {
+        return true;
+    }
+    
+    @Restricted(NoExternalUse.class)
+    @CheckForNull
+    public static Header get() {
+        List<Header> headers = ExtensionList.lookup(Header.class).stream().filter(header -> header.isHeaderEnabled()).collect(Collectors.toList());
+        if (headers.size() > 0) {
```

Comment:
```suggestion
        if (!headers.isEmpty()) {
```

Reply:
Updated in https://github.com/jenkinsci/jenkins/pull/5909/commits/88eac160e4ec72ee1a60680d1a8f36da8c79242e